rosidl_generator_py/resource/_msg.py.em

```diff
@@ -334,7 +334,19 @@ if isinstance(type_, AbstractNestedType):
         typename = self.__class__.__module__.split('.')
         typename.pop()
         typename.append(self.__class__.__name__)
-        args = [s[1:] + '=' + repr(getattr(self, s, None)) for s in self.__slots__]
+        args = []
+        for s, t in zip(self.__slots__, self.SLOT_TYPES):
+            field = getattr(self, s, None)
+            fieldstr = repr(field)
+            if isinstance(t, rosidl_parser.definition.AbstractSequence):
```

Comment:
Ug, I don't think this will work for other `AbstractSequence` types; I need to be more selective.